src/ui/app-tour/AppTour.js

```diff
@@ -406,6 +406,7 @@ class AppTour extends React.Component {
     if (!wizardEnabled || appTourProgress === APP_TOUR_CANCELLED) return null;
     const maskName = [1, 12, 14, 15].includes(appTourLastStep) ? 'Mask' : '';
     const scrollDuration = appTourLastStep === 5 ? 600 : 150;
+    const scrollLockMobile = window.innerWidth > 1000;
```

Comment:
I think `scrollLockMobile` is quite confusing since it means the opposite of what the condition (screen width > 1000) means.